### DISCUSSION TITLE
comment lines

### DISCUSSION THREAD

**Discussion by skariel** [ki-editor/ki-editor#448] [Q&A] [answered]
how to comment lines included in selection, is this possible? e.g. in nvim I use gcc or ctrl+/ in vscode (I did found how to indent lines, with ">" )

**Comment by skariel** [comment]
ok.. found the proper way to do this: select region to comment (using any selection mode): `v q e E i #`. (for python)

**Reply by wongjiahau** [reply, marked as answer]
I’m glad you found it! 

But anyway I have no plan to implement comment yet, as they are language specific.
